tests/conftest.py

```diff
@@ -578,3 +555,18 @@ def run_io_in_bg():
 
         thread = threading.Thread(target=run_io_in_bg)
         thread.start()
+
+
+@pytest.fixture(
```

Comment:
does it mean every test will have to run on both cephfs and rbd?

Reply:
no 'have to' but optional

Reply:
I find this part of code confusing? doesn't it iterate for params,  can you explain in docstring here?